src/app/donation-start/donation-start-form/donation-start-form.component.ts

```diff
@@ -1010,13 +998,30 @@ export class DonationStartFormComponent implements AfterContentChecked, AfterCon
       return;
     }
 
+    if (!this.stripeElements) {
+      console.error('Stripe Elements not ready');
+      return;
+    }
+
     // Card element is mounted the same way regardless of donation info. See
     // this.createDonationAndMaybePerson().subscribe(...) for Payment Request Button mount, which needs donation info
     // first and so happens in `preparePaymentRequestButton()`.
-    this.card = this.stripeService.getCard();
-    if (this.cardInfo && this.card) { // Ensure #cardInfo not hidden by PRB success.
-      this.card.mount(this.cardInfo.nativeElement);
-      this.card.on('change', this.cardHandler);
+    this.stripePaymentElement = this.stripeElements.create(
+        "payment",
+        {
+          terms: {
+            card: "never" // we have our own terms copy for the future payment in donation-start-form.component.html
+          },
+          fields: {
+            billingDetails: 'auto', // we could set 'never' here, but then we'd have to pass all the details in our `confirmPayment` call, in more detail than we collect.
```

Comment:
Per my other comment, I'm not sure about this – it definitely feels like the same rules ought to apply and I'd like to be 100% sure and see what error we get, before we start replacing our better-optimised address fields with the Stripe ones.

Reply:
I've seen you can go into details with this - so I've now set just postcode and country to 'never' and left all the other billing details on their default setting where Stripe asks for them if it wants to. In my tests it doesn't seem to want to.